dist/android/dimens.xml

```diff
@@ -88,16 +88,16 @@
   <dimen name="cdr_space_inset_four_x_squish_left_right">64.00dp</dimen>
   <dimen name="cdr_space_inset_four_x_stretch_top_bottom">96.00dp</dimen>
   <dimen name="cdr_space_inset_four_x_stretch_left_right">64.00dp</dimen>
-  <dimen name="cdr_text_default_display_1_line_spacing">40.00dp</dimen>
```

Comment:
The old transform is adding an extra underscore between the property name and the number when it doesn't appear in the source file. This appears to be a bug but anyone relying on the old name will have a problem.

We should probably just let our android user(s) know that we've made the change.